src/lib/components/CommunitySelectionModal/CommunitySelectionFooter.js

```diff
@@ -24,4 +24,4 @@ export const CommunitySelectionFooter = () => {
       </Container>
     </>
   );
-};
+};
```

Comment:
Why this change?

Reply:
I first committed changes to the file, but changed it back, and this happened. There is no change, so I don't know why it's showing up like this?

Reply:
its missing new line on the end of the file (please configure/run prettier)